playwright/globalSetup.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable no-console */
+
+import fs from 'fs';
+import { nextBuild } from 'next/dist/cli/next-build';
+import path from 'path';
+
+async function globalSetup() {
+    process.env.PLAYWRIGHT = '1';
+
+    // Copy env file .env.test to .env.production
+    fs.copyFile(path.join(__dirname, '..', '.env.test'), path.join(__dirname, '..', '.env.production'), (err) => {
```

Comment:
This function is asynchronous and I believe it may not completed before the next step in the setup, i.e. the `nextBuild()`. Maybe use [`fs/promises` instead of `fs`](https://nodejs.org/api/fs.html#fs_fspromises_copyfile_src_dest_mode), and `await` until `copyFile()` resolves?